jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -177,6 +177,8 @@ public static void setServerInfo(String serverInfo)
     private final Map<String, String> _initParams;
     private ClassLoader _classLoader;
     private boolean _contextPathDefault = true;
+    private String _defaultRequestCharacterEncoding;
+    private String _defaultResponseCharacterEncoding;
```

Comment:
Is there a reason these fields and methods have "default" in their name?
I would just called them without "default" as they are in `ServletContext`.

Reply:
I'm disinclined to change the name. The "default" arises from the fact that it is the default for the context. Without "default" in the name, where the getters are accessed, it gets very confusing with so many different sources of character encoding flying around, so I like that it is obvious that it is the default only.

Reply:
It is not the default for the context, it is the default for the requests or the responses. This is the only place "default" is used.
It is not used in `ServletContext`, it is not used in `web.xml`, it is not used in `StandardDescriptorProcessor`, and not even used in test method names, so I don't see any reason to change a name used in so many other places and add "default" (which really adds no additional clarity) in just this place.